source/chapters/user_interface.rst

```diff
@@ -926,8 +938,10 @@ Crossfader
    The crossfader section of the mixer
 
 The :term:`crossfader` determines the actual volume of each deck when moving
-the slider from left to right. If both decks are playing and the crossfader is
-in its default center position, then you will hear both decks. Right-clicking on
+the slider from left to right. Simply put, the crossfader aids the user to regulate the volume and the intensity of the sound output from both deck.
+Crossfading is a sound output effect that allows the user play more than one song at a time on different deck.
+It is a very useful function when making music mix. The user can slowly fadeout a song with the crossfader and allow the new song come up.
+If both decks are playing and the crossfader is in its default center position, then you will hear both decks. Right-clicking on
```

Comment:
this change is not necessary IMO.
instead add a section about crossfader assignment switches. those allow you to assign a deck to the left, right or center position of the crossfader.
you find them here:
2 decks
![image](https://user-images.githubusercontent.com/5934199/96016129-b1e5e580-0e48-11eb-9ec7-113234bc1c1e.png)
4 decks 
![image](https://user-images.githubusercontent.com/5934199/96016197-c4601f00-0e48-11eb-8bb1-3a21c0e9c4e6.png)